File: 2025-2/sesion-de-clase-03/java/pe/edu/pucp/inf30/softprog/Principal.java
package pe.edu.pucp.inf30.softprog;

import pe.edu.pucp.inf30.softprog.config.CadenaConexion;

public class Principal {
	public static void main(String[] args) {
		CadenaConexion cadenaConexion = new CadenaConexion.Builder()
			.servidor("127.0.0.1")
			.schema("inf30")
			.puerto(3306)
			.build();
			
		System.out.println(cadenaConexion);
	}
}

File: 2025-2/sesion-de-clase-03/dotNET/Principal.cs
using System;
using PUCP.Inf30.SoftProg.Config;

namespace PUCP.Inf30.SoftProg {
	public class Principal {
		public static void Main(string[] args) {
			CadenaConexion cadenaConexion = new CadenaConexion.Builder()
				.ConServidor("127.0.0.1")
				.ConSchema("INF30")
				.EnPuerto(3306)
				.Build();
				
			Console.WriteLine(cadenaConexion);
		}
	}
}
